Libraries/LibCore/CTCPServer.h

```diff
@@ -16,6 +16,9 @@ class CTCPServer : public CObject {
 
     RefPtr<CTCPSocket> accept();
 
+    IPv4Address local_address();
+    u16 local_port();
```

Comment:
Since these do not change the state of the socket, let's make them `const`:
```suggestion
    IPv4Address local_address() const;
    u16 local_port() const;
```